bin/ch-test

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+
+libexec="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+. "${libexec}/base.sh"
+. "${libexec}/version.sh"
+
+# shellcheck disable=SC2034
+usage=$(cat <<EOF
+Run the Charliecloud test suite.
+
+Usage:
+
+  $ $(basename "$0") [PHASE] [OPTIONS]
+EOF
+)
+
+check_builder () {
```

Comment:
I tend to name things most-significant-first, i.e. this would be called `builder_check` and below `set_builder` would become `builder_set`. The reasoning is that then when you alphabetize them, related functions end up near one another. I suggest making this change.